test/WebJobs.Script.Tests/EndToEndTestFixture.cs

```diff
@@ -36,6 +37,8 @@ protected EndToEndTestFixture(string rootPath, string testId)
                 FileLoggingEnabled = true
             };
 
+            ApiHubTestHelper.SetDefaultConnectionFactory();
```

Comment:
That needs to run before the config.UseApiHub call. Ideally all service classes should be injectable but they are not, so I couldn't find a better place to put it. However it should not impact other tests.